.goreleaser.yml

```diff
@@ -0,0 +1,32 @@
+project_name: wego
+builds:
+  - <<: &build_defaults
+      binary: wego
+      main: ./cmd/wego
+      ldflags:
+        - -X github.com/weaveworks/weave-gitops/cmd/wego/version.BuildTime={{.Date}}
+        - -X github.com/weaveworks/weave-gitops/cmd/wego/version.Branch=$(BRANCH) 
+        - -X github.com/weaveworks/weave-gitops/cmd/wego/version.GitCommit={{.Commit}}
+        - -X github.com/weaveworks/weave-gitops/pkg/version.FluxVersion=$(FLUX_VERSION)
+      env:
+        - CGO_ENABLED=0
+      hooks:
+        pre: export BRANCH=git rev-parse --abbrev-ref HEAD
+        pre: export CURRENT_DIR=pwd
+        pre: export FLUX_VERSION=$(CURRENT_DIR)/tools/bin/stoml $(CURRENT_DIR)/tools/dependencies.toml flux.version
```

Comment:
Will `stoml` be available here? We download it in the Makefile

Reply:
You might be right. I may need to do some stuff